sources/web/datalab/polymer/components/datalab-app/datalab-app.ts

```diff
@@ -125,6 +125,29 @@ class DatalabAppElement extends Polymer.Element {
     }
   }
 
+  /**
+   * Shows a notification toast at the bottom of the page with the given message.
+   * @param message message to show in the notification toast
+   * @param sticky whether the toast should stick around until explicitly dismissed in code
+   */
+  showNotification(message: string, sticky?: boolean) {
```

Comment:
If you are going to be using this toast for things other than connection messages, then the component should probably not be called connectionToast.